fix: fix BrowserFetcher._getFolderPath to support relative install paths

nadir opened the pull request:
This patch fixes the BrowserFetcher._getFolderPath method so that it supports relative download paths using PUPPETEER_DOWNLOAD_PATH or npm config

Issues: #7592

nadir committed:
fix: fix BrowserFetcher._getFolderPath to support relative install paths

This patch fixes the BrowserFetcher._getFolderPath method so that it supports relative download paths using PUPPETEER_DOWNLOAD_PATH or npm config

Issues: #7592

nadir commented:
@googlebot I signed it!

jschfflr commented on src/node/BrowserFetcher.ts:
```diff
@@ -419,7 +419,9 @@ export class BrowserFetcher {
    */
   _getFolderPath(revision: string): string {
-    return path.join(this._downloadsFolder, `${this._platform}-${revision}`);
+    return path.resolve(
```
You can just path multiple path pieces to `path.resolve`:
```js
path.resolve(this._downloadsFolder, `${this._platform}-${revision}`);
```

See: https://nodejs.org/api/path.html#path_path_resolve_paths

jschfflr commented:
@nadhirhb Thanks for your contribution!

nadir committed:
fix: fix BrowserFetcher._getFolderPath to support relative install paths

This patch fixes the BrowserFetcher._getFolderPath method so that it supports relative download paths using PUPPETEER_DOWNLOAD_PATH or npm config

Issues: #7592

jschfflr committed:
Merge branch 'main' into fix-relative-install-path
